docs/source/index.rst

```diff
@@ -11,30 +11,15 @@ It is a simple build tool, it does no dependency management.
 
 The recommended way to invoke is by calling the module:
 
-.. code-block:: sh
-
-   $ python -m build
-   usage: python -m build [-h] [--sdist] [--wheel] [--outdir ./dist] [--skip-dependencies] [.]
-
-   positional arguments:
-   .                     source directory (defaults to current directory)
-
-   optional arguments:
-   -h, --help            show this help message and exit
-   --sdist, -s           build a source package
-   --wheel, -w           build a wheel
-   --outdir ./dist, -o ./dist
-                           output directory
-   --skip-dependencies, -x
-                           does not check for the dependencies
+.. program-output:: python -m build --help
```

Comment:
Can we instead use https://pypi.org/project/sphinxcontrib-autoprogram/#description that generates CLI flags that can be URL linked? (see https://tox.readthedocs.io/en/latest/config.html#cli)

Reply:
Done.

On top for #22 for the isolated `main_parser` function.